iron-input.html

```diff
@@ -47,6 +56,34 @@
 @demo demo/index.html
 */
 
+  (function () {
+
+      /* Internet Explorer 11 may have trouble retrieving the number type
+      of an input value. This short script performs a quick test, and repairs
+      the functionality if necessary. Load before attempting to use the
+      `valueAsNumber` property on input elements. */
+
+      'use strict';
+
+      var a = document.createElement( 'input' );
```

Comment:
this should be executed only when iron-input.html is loaded for the first time. @cdata please confirm